doc/nrf/libraries/dfu/dfu_target.rst

```diff
@@ -141,6 +141,11 @@ To do so, enable the following options:
 
 The MCUboot target will then use the :ref:`zephyr:settings_api` subsystem in Zephyr to store the current progress used by the :c:func:`dfu_target_write` function across power failures and device resets.
 
+Using a dedicated partition for full modem upgrades
+===================================================
+
+Enable the :kconfig:option:`DFU_TARGET_FULL_MODEM_USE_EXT_PARTITION` Kconfig option to use :ref:`partition_manager` for creating a dedicated partition for full modem upgrades.
```

Comment:
Don't you also need to set some override in your DTS?

Reply:
not that i know of.

Reply:
https://github.com/nrfconnect/sdk-nrf/blob/main/subsys/partition_manager/Kconfig.template.partition_region#L1

So you need to set the chosen `nordic,pm-ext-flash` as described here https://github.com/nrfconnect/sdk-nrf/blob/main/scripts/partition_manager/partition_manager.rst#external-flash-memory-partitions

Reply:
ah, ok.  do you think there should be an explicit mention of that in the `dfu_target.rst`?

for the samples related to this PR, we do have tha required line in the overlay.

Reply:
Other users of `dfu_target` will need to enable that `chosen`, so a mention of the `chosen` and a link to where it is described how to set it would be beneficial IMO.